mentions/views.py

```diff
@@ -7,7 +7,10 @@
 
 
 def autocomplete(request):
-    term = request.GET.get('term', '')
+    # to be haystack+solr friendly we must strip and join spaces
+    term = request.GET.get('term', '').strip()
+    term = ' '.join(term.split())
```

Comment:
yes, we have it within `Provider.search()` in two places in our codebase and it's not DRY at all
that was the reason for pr
thank you!